src/ADTypes.jl

```diff
@@ -6,6 +6,7 @@
 module ADTypes
 
 using Base: @deprecate
+using Compat: @compat # for `public`
```

Comment:
There are other public names: everything that belongs to the docs page https://sciml.github.io/ADTypes.jl/stable/ is public

Reply:
Added in f8d1397, unless I missed something.